timetable/demo/timetable_demo.xml

```diff
@@ -0,0 +1,326 @@
+            <field name="name">Economic</field>
+            <field name="code">ECO</field>
+            <field name="maximum_marks">100</field>
+            <field name="minimum_marks">35</field>
+            <field name="teacher_ids" eval="[(6, 0,[ref('school.demo_school_teacher_1')])]"/>
+        </record>
+        <record id="demo_subject_subject_10" model="subject.subject">
+            <field name="name">Account</field>
+            <field name="code">ACC</field>
+            <field name="maximum_marks">100</field>
+            <field name="minimum_marks">35</field>
+            <field name="teacher_ids" eval="[(6, 0,[ref('school.demo_school_teacher_2'),ref('school.demo_school_teacher_3')])]"/>
+        </record>
+        <record id="demo_subject_subject_11" model="subject.subject">
+            <field name="name">Sanskrit</field>
+            <field name="code">SAN</field>
+            <field name="maximum_marks">100</field>
+            <field name="minimum_marks">35</field>
+            <field name="teacher_ids" eval="[(6,0,[ref('school.demo_school_teacher_1')])]"/>
+        </record>
+        <record id="school.demo_school_teacher_1" model="school.teacher">
+            <field eval="[(6,0,[ref('school.demo_subject_subject_1'),ref('school.demo_subject_subject_2'),ref('school.demo_subject_subject_3'),ref('school.demo_subject_subject_4'),ref('school.demo_subject_subject_5'),ref('demo_subject_subject_9'),ref('demo_subject_subject_11')])]" name="subject_id"/>
+        </record>
+        <record id="school.demo_school_teacher_2" model="school.teacher">
+            <field eval="[(6,0,[ref('school.demo_subject_subject_2'),ref('school.demo_subject_subject_3'),ref('school.demo_subject_subject_4'),ref('school.demo_subject_subject_5'),ref('school.demo_subject_subject_6'),ref('demo_subject_subject_10'),ref('demo_subject_subject_11')])]" name="subject_id"/>
+        </record>
+        <record id="school.demo_school_teacher_3" model="school.teacher">
+            <field eval="[(6,0,[ref('school.demo_subject_subject_1'),ref('school.demo_subject_subject_2'),ref('school.demo_subject_subject_6'),ref('demo_subject_subject_10'),ref('demo_subject_subject_11')])]" name="subject_id"/>
+        </record>
+
+        <record id="time_table_firsts0" model="time.table">
+            <!-- <field name="division_id" ref="school.demo_standard_division_2"/> -->
+            <field name="name">First</field>
+            <field name="standard_id" ref="school.demo_standard_standard_2"/>
+            <!-- <field name="medium_id" ref="school.demo_standard_medium_1"/>
+            <field name="class_id" ref="school.demo_school_standard_2"/> -->
+            <field name="year_id" ref="school.demo_academic_year_1"/>
+        </record>
+        <record id="time_table_line_0" model="time.table.line">
+            <field name="teacher_id" ref="school.demo_school_teacher_1"/>
+            <field eval="9.0" name="start_time"/>
+            <field name="week_day">monday</field>
```

Comment:
Please check with different Language, as the weekday is not working in previous version because of weekday names are different in other lang. @ForamSerpentCS

Reply:
Yeas ma'am, checked it with different language in separate database, it has installed perfectly as 'week_day' is selection field, no language issue generated with demo data installation. @AnuKPatel